.editorconfig

```diff
@@ -90,7 +90,7 @@ indent_style = unset
 insert_final_newline = unset
 trim_trailing_whitespace = unset
 
-[pkgs/development/compilers/elm/registry.dat]
+[registry.dat]
```

Comment:
btw this file is a binary which includes metadata for all the packages listed in https://package.elm-lang.org/ in theory it could be shared across all items packaged with elm2nix. So it depends if we would rather have one which sometimes needs to be updated (to maybe get some newly published packages) or if every package should include its own which means there would be some duplicated data spread around the repo.